docs/advanced_topics/embeds.md

```diff
@@ -0,0 +1,409 @@
+fetching embed code using [Embedly](Embedly) and [custom embed finders](custom_embed_finders).
+
+## Embedding content on your site
+
+Wagtail's embeds module should work straight out of the box for most providers.
+You can use any of the following methods to call the module:
+
+### Rich text
+
+Wagtail's default rich text editor has a "media" icon that allows embeds to be
+placed into rich text. You don't have to do anything to enable this; just make
+sure the rich text field's content is being passed through the ``|richtext``
+filter in the template as this is what calls the embeds module to fetch and
+nest the embed code.
+
+### `EmbedBlock` StreamField block type
+
+The `EmbedBlock` block type allows embeds
+to be placed into a `StreamField`.
+
+The `max_width` and `max_height` arguments are sent to the provider when fetching the embed code.
+
+For example:
+
+```python
+from wagtail.embeds.blocks import EmbedBlock
+
+class MyStreamField(blocks.StreamBlock):
+    ...
+
+    embed = EmbedBlock(max_width=800, max_height=400)
+```
+
+### `{% embed %}` tag
+
+Syntax: `{% embed <url> [max_width=<max width>] %}`
+
+You can nest embeds into a template by passing the URL and an optional
+`max_width` argument to the `{% embed %}` tag.
+
+The `max_width` argument is sent to the provider when fetching the embed code.
+
+```html+Django
+{% load wagtailembeds_tags %}
+
+{# Embed a YouTube video #}
+{% embed 'https://www.youtube.com/watch?v=Ffu-2jEdLPw' %}
+
+{# This tag can also take the URL from a variable #}
+{% embed page.video_url %}
+```
+
+### From Python
+
+You can also call the internal `get_embed` function that takes a URL string
+and returns an `Embed` object (see model documentation below). This also
+takes a `max_width` keyword argument that is sent to the provider when
+fetching the embed code.
+
+```python
+from wagtail.embeds.embeds import get_embed
+from wagtail.embeds.exceptions import EmbedException
+
+try:
+    embed = get_embed('https://www.youtube.com/watch?v=Ffu-2jEdLPw')
+
+    print(embed.html)
+except EmbedException:
+    # Cannot find embed
+    pass
+```
+
+(configuring_embed_finders)=
+
+## Configuring embed "finders"
+
+Embed finders are the modules within Wagtail that are responsible for producing
+embed code from a URL.
+
+Embed finders are configured using the `WAGTAILEMBEDS_FINDERS` setting. This
+is a list of finder configurations that are each run in order until one of them
+successfully returns an embed:
+
+The default configuration is:
+
+```python
+WAGTAILEMBEDS_FINDERS = [
+    {
+        'class': 'wagtail.embeds.finders.oembed'
+    }
+]
+```
+
+(oEmbed)=
+
+### oEmbed (default)
+
+The default embed finder fetches the embed code directly from the content
+provider using the oEmbed protocol. Wagtail has a built-in list of providers
+which are all enabled by default. You can find that provider list at the
+following link:
+
+<https://github.com/wagtail/wagtail/blob/main/wagtail/embeds/oembed_providers.py>
```

Comment:
This link may not work, please check it.

Reply:
Hi @lb-, that link is unchanged since 2.14 and still works. It didn’t appear in any of the before/after diffs.